apps/sms/index.html

```diff
@@ -148,6 +155,33 @@ <h1 data-l10n-id="messages"></h1>
           <a id="threads-settings-button" data-icon="settings" data-l10n-id="settingsButton"></a>
         </gaia-header>
         <article id="threads-container" class="view-body edit-container" data-type="list">
+          <gaia-fast-list id="threads-list" top="0" bottom="0" caching>
+            <!--<template header>
+              <gaia-sub-header data-time="${daySection}">${section}</gaia-sub-header>
+            </template>-->
+            <template item>
```

Comment:
Wondering why list can't handle that, but anyway would be nice to put such limitations to the fast-list README.md :)

Regarding language-change, although it's quite inconvenient we can probably keep all data required for localization inside model and update everything on language change..... I don't really like it to be honest as I'd rather have data model without any visual-related things, but we can live with it if it's too hard to fix from fast-list side.